File: Praktikum_Pemrograman1-Modul 5/C/PRAK504-2310817320006-Alya_Rosaan.c
#include <stdio.h>
int reverse(int angka){
    int reversed = 0;
    int sisa;
    while (angka > 0){;
        sisa = angka % 10;
        reversed = reversed * 10 + sisa;
        angka /= 10;}
    return reversed;
    
}
        int main(){
            int A, B;
            scanf("%d %d", &A, &B);
            A = reverse(A);
            B = reverse(B);
            int C = A + B;
            printf("%d", reverse(C));
        }

File: Praktikum_Pemrograman1-Modul 5/PYTHON/PRAK504-2310817320006-Alya_Rosaan.py
def reverse(angka):
    reversed = 0
    while int(angka) > 0:
        sisa = int(angka) % 10
        reversed = reversed * 10 + sisa
        angka /= 10
    return reversed
    
A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))
        